src/sdk/clients/decorators/erc7579/index.ts

```diff
@@ -146,6 +146,7 @@ export function erc7579Actions() {
   })
 }
 
+// Review: if this should be imported from module-sdk
 export type Module = {
```

Comment:
Yep, we can import this from module-sdk, should be removed from here.

Reply:
tried but other files are complaining about incompatible types. keeping review comment and this custom type for now